src/symmetricDifferenceWith.js

```diff
@@ -0,0 +1,31 @@
+var _curry3 = require('./internal/_curry3');
+var converge = require('./converge');
+var differenceWith = require('./differenceWith');
+var flip = require('./flip');
+var unionWith = require('./unionWith');
+
+
+/**
+ * Finds the set (i.e. no duplicates) of all elements contained in the first or second list, but not both.
+ * Duplication is determined according to the value returned by applying the supplied predicate to two list
+ * elements.
```

Comment:
@davidchambers Will do. Just so I'm clear, you're only talking about the docstrings, correct?